Dockerfile

```diff
@@ -132,10 +142,10 @@ RUN set -ex \
           libxslt-dev \
           libzip-dev \
           postgresql-dev \
+          oniguruma-dev \
     && docker-php-ext-configure gd \
-                --with-freetype-dir=/usr/include/ \
-                --with-jpeg-dir=/usr/include/ \
-                --with-png-dir=/usr/include/ \
+                --with-freetype \
+                --with-jpeg \
```

Comment:
just curious about why these changed: do we no longer need to specify a dir for these? will dir be the same for backwards compatibility? why don't we need to specify png any more?

Reply:
php 7.4 changed these flags, and png is now supported by default - https://www.php.net/manual/en/migration74.other-changes.php

Reply:
will these still default to /usr/include? I suppose it doesn't matter, since this is for OpenRESTY internals and not really for apps using OpenRESTY, so as long as OpenRESTY can find it's own includes...